products/kyverno.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Kyverno
+category: server-app
+tags: cncf
+permalink: /kyverno
+alternate_urls:
+-   /kyverno
+versionCommand: |-
+  kyverno version
+releasePolicyLink: https://kyverno.io/docs/releases/
```

Comment:
https://kyverno.io/docs/releases/ does not mention any support guarantees so we switch to something that does.

```suggestion
releasePolicyLink: https://kyverno.io/docs/installation/#compatibility-matrix
```